{
  "@type": "NewsArticle",
  "identifier": "2022--06--06--en--myfeed--HackerNews--NewsArticle--31642797",
  "url": "https://news.ycombinator.com/item?id=31642797",
  "headline": "Raising exceptions or returning error objects in Python",
  "publisher": {
    "@type": "Organization",
    "name": "HackerNews",
    "url": "https://news.ycombinator.com",
    "logo": "https://hn.buzzing.cc/avatar.png"
  },
  "description": "",
  "keywords": [],
  "author": {
    "@type": "Person",
    "name": "tea_drinker",
    "url": "https://news.ycombinator.com/user?id=tea_drinker"
  },
  "discussionUrl": "https://news.ycombinator.com/item?id=31642797",
  "sameAs": "https://lukeplant.me.uk/blog/posts/raising-exceptions-or-returning-error-objects-in-python/",
  "dateCreated": "2022-06-06T17:12:02.710Z",
  "datePublished": "2022-06-06T16:49:20.000Z",
  "dateModified": "2022-06-06T17:12:02.710Z",
  "interactionStatistic": [
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "LikeAction"
      },
      "userInteractionCount": 2
    },
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "CommentAction"
      },
      "userInteractionCount": 0
    }
  ],
  "headline_zh-Hans": "在Python中引发异常或返回错误对象\n",
  "headline_zh-Hant": "在Python中引發異常或返回錯誤對象\n",
  "@context": [
    "https://schema.org",
    {
      "@vocab": "http://schema.org/",
      "@language": "en",
      "headline_zh-Hans": {
        "@id": "headline",
        "@language": "zh-Hans"
      },
      "headline_zh-Hant": {
        "@id": "headline",
        "@language": "zh-Hant"
      },
      "@version": 1.1
    }
  ]
}